pytest.ini

```diff
@@ -11,3 +11,5 @@ addopts =
 testpaths =
     test
 xfail_strict = True
+markers =
+    opcheck_only_one: only opcheck one parametrization
```

Comment:
For a reason that I don't remember, we define `pytest` markers in [`conftest.py` instead](https://github.com/pytorch/vision/blob/main/test/conftest.py#L17).


But what is the purpose of this marker? I don't see it being handled anywhere and I don't see it in the `pytorch` core repo either.

Reply:
It's tested for at torch/testing/_internal/optests/generate_tests.py and says "if this test is parametrized, use only one parametrization when generating the opcheck test". This makes the opcheck tests run a lot faster since we aren't, e.g., rerunning them under different seeds, which is usually pointless